### DISCUSSION TITLE
How do I build faster-whisper against Cuda 12.2?

### DISCUSSION THREAD

**Discussion by Geremia** [SYSTRAN/faster-whisper#388] [Q&A] [answered]
How do I build faster-whisper from scratch against Cuda 12.2?

**Comment by phineas-pta** [comment, marked as answer]
u need compile `ctranslate2`

https://github.com/OpenNMT/CTranslate2/blob/master/docs/installation.md

**Comment by Geremia** [comment]
@phineas-pta And then how do I build faster-whisper from source?

**Reply by guillaumekln** [reply]
Once your custom CTranslate2 build is installed, you can install faster-whisper normally with `pip install faster-whisper`.
